caf-audit-service-container/README.md

```diff
@@ -22,22 +22,11 @@ The number of replica shards (copies) that each primary shard should have. e.g.
 
 The path to the directory containing the config.properties which can be used as an alternative means of specifying the environment variables. e.g. .\config-props
 
-#### Logging
+##### CAF\_LOG\_LEVEL
```

Comment:
Change section to:

##### CAF\_LOG\_LEVEL

The logging level for the CAF Audit Web Service. e.g debug

Logging levels supported include:

* warn
* error
* info
* trace
* debug
* all

A default logging level of **info** is applied if this environment variable is not configured.